ee/api/billing.py

```diff
@@ -153,7 +153,17 @@ def list(self, request: HttpRequest, *args: Any, **kwargs: Any) -> Response:
             not billing_service_response.get("customer", {}).get("has_active_subscription")
             and not settings.BILLING_V2_ENABLED
         ):
-            if not (distinct_id and posthoganalytics.get_feature_flag("billing-v2-enabled", distinct_id)):
+            if not (
+                distinct_id
+                and org
+                and posthoganalytics.get_feature_flag(
+                    "billing-v2-enabled",
+                    distinct_id,
+                    groups={
+                        "organization": str(org.id),
+                    },
```

Comment:
optional, but since you already have this, you might as well pass 

```suggestion
                    },
                    group_properties={
                    "organization": {
                        "id": str(self.organization.id),
                        "created_at": self.organization.created_at,
                    }
                },
```

which can speed things up decently